sealtk/core/VideoMetaData.cpp

```diff
@@ -0,0 +1,77 @@
+/* This file is part of SEAL-TK, and is distributed under the OSI-approved BSD
+ * 3-Clause License. See top-level LICENSE file or
+ * https://github.com/Kitware/seal-tk/blob/master/LICENSE for details. */
+
+#include <sealtk/core/VideoMetaData.hpp>
+
+namespace kv = kwiver::vital;
+
+namespace sealtk
+{
+
+namespace core
+{
+
+// ============================================================================
+class VideoMetaDataData : public QSharedData
```

Comment:
Yikes... that is an unfortunate class name :)

Reply:
Eh, it happens :slightly_smiling_face:.